src/main/java/frc/robot/subsystems/intake/IntakeIOSim.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix6.controls.PositionVoltage;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Velocity;
+import lib.motors.SparkMaxSim;
+import lib.motors.TalonFXSim;
+
+public class IntakeIOSim implements IntakeIO {
+    private TalonFXSim angleMotor;
+    private SparkMaxSim frontRoller;
+    private SparkMaxSim centerRoller;
+    private PositionVoltage positionRequest = new PositionVoltage(0);
```

Comment:
- "position request" is too general, find a more specific name.
- Should use Motion Magic Expo.
- Consult with @vichik123, but I think we should use closed-loop control in all of this subsystem's motors (velocity for the rollers motors). It's not a lot of additional work but will make our intaking more accurate (and it's bonus points from judges).

Reply:
Seems a little excessive.

Reply:
The only added work is to run SysId and set the gains in code. I'd say, try to tune it for 30 minutes max, if it doesn't work revert to percent output.

Reply:
And what about the other things?